main.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import os.path
+from argparse import Namespace
+
+ERR_MSG = "Error message: Invalid input. Make sure the input file is a valid csv file."
+
+
+def get_args() -> Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input_file", type=str, required=True)
+    parser.add_argument("-o", "--output_file", type=str, required=True)
+    parser.add_argument("-r", "--row_limit", type=int, required=True)
+    return parser.parse_args()
+
+
+def is_input_valid(input_file: str, row_limit: int) -> bool:
+    is_valid_file = os.path.isfile(input_file) and os.path.exists(input_file)
+    is_csv = input_file.lower().endswith(".csv")
+    if not (is_valid_file and is_csv):
+        return False
+
+    with open(input_file, "r") as fp:
```

Comment:
`fp` is not a meaningful variable name. I cannot understand simply from the name what's its purpose.
See https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c.